src/containers/PatientProfilePage.tsx

```diff
@@ -228,12 +229,15 @@ const PatientProfilePage = ({
             }}
           />
           <RadioSelector
-            labels={['Male', 'Female']}
+            labels={Object.keys(Gender)}
             currentValue={formFields.gender}
             handleChange={(e: React.ChangeEvent<HTMLElement>) => {
               setFormFields({
                 ...formFields,
-                gender: (e.target as HTMLInputElement).value,
+                gender:
+                  (e.target as HTMLInputElement).value === Gender.M
+                    ? Gender.M
+                    : Gender.F,
```

Comment:
isn't this validation unnecessary? since we're getting this value from the input field (which is a radio button), it should be guaranteed that we're only getting `'M'` or `'F'` as values back
```suggestion
                  (e.target as HTMLInputElement).value
```

Reply:
That's correct but we need to parse the string return as an enum value. I tried this change and we couldn't make the `gender` field equal to `e.target.value`.